doc/doc_utils/jumpstart_doc_utils.py

```diff
@@ -0,0 +1,87 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+from urllib import request
+import json
+from packaging.version import Version
+
+JUMPSTART_REGION = "eu-west-2"
```

Comment:
non-blocking: if you have time, please add a comment as to why we use `DUB` so that someone doesn't just change it back to `us-west-2`. Also, could you rename `JUMPSTART_METADATA_REGION`?

Reply:
+1